Sources/StreamChat/Controllers/MessageController/MessageController.swift

```diff
@@ -55,6 +55,7 @@ public class ChatMessageController: DataController, DelegateCallable, DataStoreP
     /// To observe changes of the reactions, set your class as a delegate of this controller or use the provided
     /// `Combine` publishers.
     ///
+    // TODO: Why is this not an observer?
     public var reactions: [ChatMessageReaction] = [] {
```

Comment:
Our reactions in `ChatMessageController` are not using an observer. Which means, it is not really offline-first. But for now, it is okay. It makes things simpler and it does not seem to require any sorting on CoreData, nor in the local memory data.